app/components/c-header-links.vue

```diff
@@ -0,0 +1,168 @@
+<template>
+  <div :class="b()">
+    <div v-for="item in items"
+         v-if="item.loggedIn == userLoggedIn"
+         :class="b('item', { active: item.isActive })"
+         :key="item.link"
+    >
+      <a :class="b('link')" :href="$t(item.link)">
+        <e-icon
+          :class="b('icon')"
+          :icon="item.icon"
+          inline
+          width="25px"
+          height="25px"
+          @click="onClick"
+        />
+        <span v-if="item.label" :class="b('label')">{{ $t(item.label) }}</span>
+        <span v-else :class="b('label')">{{ username }}</span>
```

Comment:
@adrianburtscher @claudio147 is right. This could cause problems. But I would recommend to switch the cases. `username` is a special case, while `item.label` is the default. Then check if `item.label` exists before calling `$t()`.